pyproject.toml

```diff
@@ -53,7 +53,7 @@ sphinx = "^2.2"
 recommonmark = "^0.6.0"
 sphinx-rtd-theme = "^0.4.3"
 babel = "^2.7.0"
-BabelGladeExtractor = "^0.5.1"
+babelgladeextractor = { git = "https://github.com/gaphor/babel-glade.git", branch = "master" }
```

Comment:
It looks like poetry (including 1.0.0b5) is currently broken when trying to install from a git repo, I think that is why you are getting the errors in Windows with CI.